src/plots.py

```diff
@@ -151,27 +152,43 @@ def plot_histo(
         if layered
         else {}
     )
-
     st.vega_lite_chart(
         data=df,
         spec={
             **CONFIG_MAIN,
             "mark": {"type": "bar", **CONFIG_MARK, "blend": "normal"},
             "params": config_params,
+            "transform": [
+                {"bin": True, "field": col_x, "as": "bin_col_x", "maxbins": bin},
```

Comment:
I would suggest the following to make normalized histogram responsible to the bin slider

```suggestion
                {"bin": config_bin, "field": col_x, "as": "bin_col_x"},
```

Reply:
Hey! made this change, and the slider is responsive with normalize checked now. That must mean binned and maxbins cannot be set simultaneously.